change/@fluentui-react-examples-674a253d-3d6d-474c-bacd-63f704444287.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Please delete this file since this package is no longer published

Reply:
Thanks @ecraig12345.